Extensions/autoSkipFestive.js

```diff
@@ -0,0 +1,35 @@
+// @ts-check
+
+// NAME: Non-festive Spotify
+// AUTHOR: Zedeldi (based on https://github.com/khanhas/spicetify-cli/blob/master/Extensions/autoSkipExplicit.js)
+// VERSION: 1.0
+// DESCRIPTION: Auto skip festive songs. Toggle in Profile menu.
+
+/// <reference path="../globals.d.ts" />
+
+(function NonFestiveSpotify() {
+    if (!Spicetify.LocalStorage) {
+        setTimeout(NonFestiveSpotify, 1000);
+        return;
+    }
+
+    let isEnabled = Spicetify.LocalStorage.get("NonFestiveMode") === "1";
+
+    new Spicetify.Menu.Item("Non-festive mode", isEnabled, (self) => {
+        isEnabled = !isEnabled;
+        Spicetify.LocalStorage.set("NonFestiveMode", isEnabled ? "1" : "0");
+        self.setState(isEnabled);
+    }).register();
+
+    Spicetify.Player.addEventListener("songchange", () => {
+        if (!isEnabled) return;
+        const data = Spicetify.Player.data || Spicetify.Queue;
+        if (!data) return;
+
+        const regexList = [/christmas/i, /xmas/i, /santa/i, /claus/i, /mistletoe/i, /winter wonderland/i, /rudolph/i, /feliz navidad/i];
```

Comment:
Could you please filter `noel` or `noël` for the French? ;)

Reply:
Done! :+1: 

Unfortunately, there are always going to be false positives, and missed Christmas songs somewhere, but any further suggestions are more than welcome. Thanks.

Reply:
Thanks a lot